apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/parser/sdl/GraphSDLSchemaParser.kt

```diff
@@ -271,7 +445,7 @@ internal object GraphSDLSchemaParser {
       enumValue() != null -> enumValue().name().text
       listValue() != null -> listValue().value().map { it.parse() }
       objectValue() != null -> text
-      stringValue() != null -> text
+      stringValue() != null -> text.removePrefix("\"").removeSuffix("\"")
```

Comment:
I'm still not 100% sure about coercing rules for default values but I think the quotes are not needed here.

Reply:
Hm, yeah `text` in this context will return string with quotes, cos:
```
stringValue
  : STRING
  | BLOCK_STRING
  ;

STRING: '"' CHARACTER* '"';
BLOCK_STRING:   '"""' .*? '"""';
```